assets/js/modules/sign-in-with-google/components/settings/SettingsForm.js

```diff
@@ -62,6 +89,58 @@ export default function SettingsForm() {
 					<Cell size={ 12 }>
 						<OneTapToggle />
 					</Cell>
+					<Cell size={ 12 }>
+						<SettingsGroup
+							title={ __(
+								'Users can create new accounts',
+								'google-site-kit'
+							) }
+						>
+							<p>
+								{ anyoneCanRegister
+									? sprintf(
+											/* translators: %s: Sign in with Google service name */
+											__(
+												'Users can create new accounts on this site using %s. ',
+												'google-site-kit'
+											),
+											_x(
+												'Sign in with Google',
+												'Service name',
+												'google-site-kit'
+											)
+									  )
+									: sprintf(
+											/* translators: %s: Sign in with Google service name */
+											__(
+												'Only existing users can use %s to access their accounts. ',
+												'google-site-kit'
+											),
+											_x(
+												'Sign in with Google',
+												'Service name',
+												'google-site-kit'
+											)
+									  ) }
+								{ hasCapability &&
```

Comment:
I think the capability check here was intended to be a cap check for the capability needed for a network admin to edit the site since all SK admins who can view the module settings should by definition be able to edit any settings on the site. I'm not against keeping the cap check here, but I think we should be able to link users to the respective `wp-admin/network/settings.php` screen in a multisite context as well.